activerecord/test/cases/adapters/postgresql/json_test.rb

```diff
@@ -33,6 +33,11 @@ def test_deserialize_with_array
     x.reload
     assert_equal ["foo" => "bar"], x.objects
   end
+
+  def test_noname_columns_of_different_types_with_arrow_operator_and_double_arrow_operator
```

Comment:
We can use just `test_noname_columns_of_different_types` as a name here, imo.

Reply:
Renamed test. Thank you!